Develop

pzwolenik opened the pull request:
[DEMO LINK](https://pzwolenik.github.io/layout_miami/)

nazarmatsevych reviewed (changes requested):
Good work but demo link doesn't work, did you run "npm run deploy" command?

If you have any questions - feel free to ask them in fe_chat

pzwolenik commented:
> Good work but demo link doesn't work, did you run "npm run deploy" command?
> 
> If you have any questions - feel free to ask them in fe_chat

Hi! Yes, you're absolutly right - I correct this in new pull request. 
Also, can you tell me, what i did wrong with mapping scss to css? I got several errors with not getting empty line in styles.css - I corrected it manually, not automatically, so it took some time. 
I try postcss (non working) and prettier (didn't do antything), same as `sass --style expanded`. Is it any way to do it faster?

nazarmatsevych reviewed (changes requested):
1. clear form fields on button click
![image](https://github.com/mate-academy/layout_miami/assets/80700247/fb8bf982-9c3a-476d-b8c6-b34f70c7174d)

2. Add hover effects and cursor: pointer to all interactive elements

pzwolenik commented:
> 1. clear form fields on button click
>    ![image](https://user-images.githubusercontent.com/80700247/282577970-fb8bf982-9c3a-476d-b8c6-b34f70c7174d.png)
> 2. Add hover effects and cursor: pointer to all interactive elements

Hi!
Thank you for suggestions :) 
Ad. 1 - I changed a little function at the end of _body_ and I checked in three different browsers. Right now it works as you asked :) 
Ad. 2 - I added to hover a different cursor.

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,384 @@
+        </a>
+
+        <div class="navigation__block">
+          <a href="tel:+1 234 555-55-55" class="phone">
```
Need to fix the format phone everywhere only in href
```suggestion
          <a href="tel:+12345555555" class="phone">
```

volodymyr-soltys97 reviewed (changes requested):
Still not fixed from the previous review
1. Now I see that you turned off autocomplete, but this is the wrong decision, it does not need to be turned off, but you need to style it additionally, you can see how to do it here 
https://css-tricks.com/snippets/css/change-autocomplete-styles-webkit-browsers/
<img width="1315" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/b178cff5-66f2-4fd1-8fa0-c0b1014dada7">

2. The form is now not cleared after the submit button, if you have any questions feel free in the fe_chat
<img width="536" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/1a798145-a229-44db-9371-07ed77b35079">

pzwolenik commented:
Hi! I changed function `submitForm()` and added to form attributes _action_ and _method_. I still didn't see it on GitHub - I'll be honest, I'm out of ideas, and I consulted the matter on the chat.
Any small hints? :)

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,371 @@
+            action="#"
+            method="post"
+            id="myForm"
+            class="form section__form contact__form"
+          >
```
```suggestion
            class="form section__form contact__form"
            onsubmit="this.reset(); return false;"
          >
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,371 @@
+      }
+    </script>
+
+    <script>
+      window.addEventListener('hashchange', () => {
+          if (window.location.hash === '#menu') {
+              document.body.classList.add('page__body--with-menu');
+          } else {
+              document.body.classList.remove('page__body--with-menu');
+          }
+      });
+    </script>
```
Move it to separate file

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,371 @@
+
+    <footer class="footer"></footer>
+
+    <script>
+      function submitForm() {
+        event.preventDefault();
+        document.getElementById("myForm").reset();
+      }
+    </script>
```
You can just remove it

volodymyr-soltys97 reviewed (changes requested):
Still not fixed from the previous review

volodymyr-soltys97 reviewed (changes requested):
Almost done!
1. You need to fix paddings for the content on the mobile version
<img width="510" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/e6237ac0-e4f6-41a8-861c-029d8c989d94">

2. Need to hide the phone number on the mobile version
<img width="162" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/f3343769-4727-458c-ae80-a023ab6499ee">

volodymyr-soltys97 reviewed (changes requested):
Still not fixed the indents from the previous review on the mobile and tablet version
To summarize, the right and left indents for all content should be smaller, currently they are too large and do not match the design indents
Demo
<img width="307" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/598cca4b-1229-4cab-9214-5660d1f40b9d">
<img width="511" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/3fc15ccf-5974-4228-b204-7b439fc9b9fc">

Design
<img width="472" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/cec49a11-a4b0-452e-8874-7f7d13551750">
<img width="849" alt="image" src="https://github.com/mate-academy/layout_miami/assets/64750958/f3845d37-a7ea-418b-bb38-fdc053750807">

pzwolenik commented:
> Still not fixed the indents from the previous review on the mobile and tablet version To summarize, the right and left indents for all content should be smaller, currently they are too large and do not match the design indents Demo <img alt="image" width="307" src="https://user-images.githubusercontent.com/64750958/283982182-598cca4b-1229-4cab-9214-5660d1f40b9d.png"> <img alt="image" width="511" src="https://user-images.githubusercontent.com/64750958/283982323-3fc15ccf-5974-4228-b204-7b439fc9b9fc.png">
> 
> Design <img alt="image" width="472" src="https://user-images.githubusercontent.com/64750958/283982198-cec49a11-a4b0-452e-8874-7f7d13551750.png"> <img alt="image" width="849" src="https://user-images.githubusercontent.com/64750958/283982304-f3845d37-a7ea-418b-bb38-fdc053750807.png">

Sorry, but I don't get it - I follow the instructions in the bunch of movies on the Platform and it should be 3 versions of layout: for width 320px , 744px and 1260px. My nav isn't dynamic, it works fine for that dimensions I wrote earlier. 

320px width: 
![image](https://github.com/mate-academy/layout_miami/assets/137695230/062576c6-973b-4c09-924e-450e2e489782)
And grid: 
![image](https://github.com/mate-academy/layout_miami/assets/137695230/53439abb-cfaa-4ec8-a2e5-707ddfc63021)

744px width: 
![image](https://github.com/mate-academy/layout_miami/assets/137695230/f62166c2-24c8-47ca-81a8-c719901b6dc8)
And grid: 
![image](https://github.com/mate-academy/layout_miami/assets/137695230/3c888cc6-0f6b-4f63-bba9-1675c6adec9e)

I can change it, sure, but it was not on the instructions - why wasn't this pointed out earlier, on the platform?

nazarmatsevych reviewed (approved):
Looks great 👍
